.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+on:
+  pull_request:
+    paths:
+      - 'src/**'
+      - '.github/workflows/test.yml'
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'latest'
+      - run: npm install
+      - run: npx vitest src/components
```

Comment:
Should there be a line break after this line?

Reply:
Doesn't need to be. More a matter of stylistic preference and readability.